test/bdd/features/oidc4vc_api.feature

```diff
@@ -7,30 +7,44 @@
 @all
 @oidc4vc_rest
 Feature: OIDC4VC REST API
-  Scenario: Credential issuance using OIDC4VC authorization code flow
-    Given issuer with id "bank_issuer" authorized as a profile user
-    And  client registered as a public client to vcs oidc
 
-    When issuer initiates credential issuance using authorization code flow
-    Then initiate issuance URL is returned
-
-    When client requests an authorization code using data from initiate issuance URL
-    And user authenticates on issuer IdP
-#     And user gives a consent to release claim data
-    Then client receives an authorization code
-
-    When client exchanges authorization code for an access token
-    Then client receives an access token
-
-  Scenario: Credential issuance using OIDC4VC pre-authorization code flow
-    Given issuer with id "bank_issuer" wants to issue credentials to his client with pre-auth code flow
-
-    When issuer sends request to initiate-issuance
-    Then issuer receives response with oidc url
-    And issuer represent this url to client as qrcode
-
-    When client scans qrcode
-    Then client should receive access token for further interactions with vc api
-
-#    When client requests credential for claim data
-#    Then client receives a valid credential
+  Background:
+    Given Organization "test_org" has been authorized with client id "test_org" and secret "test-org-secret"
+    And   Issuer with id "bank_issuer" is authorized as a Profile user
+    And   Issuer registers Client for vcs oidc interactions
+    And   User creates the wallet
+
+  Scenario: OIDC credential issuance and verification
+    When Issuer initiates credential issuance using authorization code flow
+    Then Issuer receives initiate issuance URL
+
+    When User interacts with Wallet to initiate OIDC credential issuance
+    Then Wallet receives an access token
+
+#    When Wallet requests credential for claim data using access token
+#    Then Wallet receives a valid credential
+
+    And   New verifiable credentials is created from table:
```

Comment:
I guess that this is temporary until `/oidc/credential` endpoint gets ready, perhaps it makes sense to remove all entries from the table except one - no need to run tests with all those profiles

Reply:
correct! done